### ISSUE TITLE
Quick Search Bar

### ISSUE THREAD

**Issue by sotch-pr35mac** [sotch-pr35mac/syng#53]
Include a quick search bar mini mode that would could be activated to float ontop of all other open windows on the desktop. Similar to iTunes old mini-mode.

**Comment by adrianhartanto0** [comment]
Hey! Can I work on this?

**Comment by sotch-pr35mac** [comment]
Yeah definitely! That would be great. I can put together a getting starting guide that explains the project structure and everything for you if you like.

**Comment by sotch-pr35mac** [comment]
@adrianhartanto0 Here is a link to the getting started guide: https://github.com/sotch-pr35mac/syng/wiki/Getting-Started

**Comment by adrianhartanto0** [comment]
Thanks! Will start ASAP

**Comment by sotch-pr35mac** [comment]
Great! Feel free to reach out if you have any questions
